packages/renderer/src/lib/kube/pods/PodUI.ts

```diff
@@ -16,11 +16,11 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
+import type { KubernetesObjectUI } from '../../objects/KubernetesObjectUI';
 import type { PodInfoContainerUI } from '../../pod/PodInfoUI';
 
-export interface PodUI {
-  name: string;
-  status: string;
+export interface PodUI extends KubernetesObjectUI {
+  kind: 'pods';
```

Comment:
If we want to double-check that the `KubernetesObject` being transformed to `KubernetesObjectUI` is of the appropriate kind (when #10916 is fixed), I think we would need to have `Pod` here, which is the value in the `kind` field of Kubernetes pods.

Reply:
I prefer that, but would break the url scheme. Some options are:
- change all Kubernetes urls to use the official kind :)
- have a big case statement in KubernetesColumnName
- change / augment navigation api to have something like:
```
handleNavigation({
    page: NavigationPage.KUBERNETES_RESOURCE,
    parameters: {
      kind: 'pod',
      name: 'pod-name',
      namespace: 'default',
    },
  });
```

I think I like the last option regardless. Thoughts?

Reply:
CRD (and native resources) define 2 names: `kind` and `plural` (see https://kubernetes.io/docs/reference/kubernetes-api/extend-resources/custom-resource-definition-v1/#CustomResourceDefinitionSpec).

`kind` is used for serialization, while `plural` is used in urls. I would prefer to keep this logic in PD: using `pods` in URL, while having `Pod` in the kind field of the objects.

It's not clear to me why the value of this `kind` field is related to the url? (I find acceptable to pass, when necessary, these 2 values as parameter (kind and plural)